src/scripts/core/cartesian.js

```diff
@@ -331,9 +333,27 @@ var cartesian = function () {
         },
 
         renderXAxis: function () {
-            var xAxis = this.xAxis();
-            var y = this.options.chart.plotHeight + this.options.chart.padding.top;
-            var x = this.options.chart.internalPadding.left;
+            const xAxis = this.xAxis();
+            const plotHeight = this.options.chart.plotHeight;
+            const paddingTop = this.options.chart.internalPadding.top;
+            const x = this.options.chart.internalPadding.left;
+            const animOpt = this.options.chart.animations;
+            let y = this.options.chart.padding.top;
+            const t = d3.transition().duration(this._animationDuration());
+
+            const adjustTickLabels = () => (selection) => {
+                if (this.options.xAxis.zeroPlane) {
+                    const adj = plotHeight - y + paddingTop;
+                    const textSel = selection.selectAll('.x.axis text');
+                    textSel.attr('transform', 'translate(0, ' + (adj) + ')');
+                }
+            };
+
+            if (this.options.xAxis.zeroPlane) {
```

Comment:
should probably allow this to be a function as well, for consistency.

Reply:
same here